packages/next/build/webpack-config.ts

```diff
@@ -74,9 +74,15 @@ function parseJsonFile(filePath: string) {
   try {
     return JSON5.parse(contents)
   } catch (err) {
+    if (!(err instanceof Error)) throw err
```

Comment:
How about throw if it’s missing lineNumber or columnNumber?

Reply:
I defaulted it to 0, so you know at least the file when it's missing